src/lib/eo/eo_api.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _EFL_EO_API_H
+#define _EFL_EO_API_H
+
+#ifdef EO_API
+#error EO_API should not be already defined
+#endif
+
+#define EOLIAN
+
+#ifdef _WIN32
+# ifndef EO_STATIC
```

Comment:
@felipealmeida, It seams that `EO_STATIC` is never defined, where and how it should be settled to build Eo statically?
And another thing isn't this letting `EFL one` out?